nixos/modules/virtualisation/containers.nix

```diff
@@ -518,6 +518,7 @@ in
         KillSignal = "WINCH";
 
         DevicePolicy = "closed";
+        DeviceAllow = "/dev/net/tun rw";
```

Comment:
I think preventing containers from accessing device files by default using a strict `DevicePolicy` (closed or strict) and explicitly whitelisting devices that are "safe" to use is a good choice here. Additional devices can be allowed in subsequent PRs, of course.

Personally, I'm not sure how safe it is to use FUSE in containers (but I don't see an obvious problem with it).